recursio-client/Global/PostProcess.gd

```diff
@@ -3,53 +3,53 @@ extends Node
 var _post_process_tool_scene = preload("res://addons/PostProcessingTool/PostProcessing_tool.tscn")
 
 var _pp
-var _tween
 
-#glitch
-var glitch :bool = false setget set_glitch, get_glitch
-# vignette
-var vignette :bool = false setget set_vignette, get_vignette
-var vignette_softness: float = 1.0 setget set_vignette_softness, get_vignette_softness
+# Glitch
+var glitch :bool = false setget set_glitch
+
+# Vignette
+var vignette :bool = false setget set_vignette
```

Comment:
I mean I know they aren't necessary yet, but why did you remove all getter?

Reply:
I didn't see any possible reason why we'd want to do something specific in a getter other than returning the variable, or is there something you had in mind?

Reply:
Not sure, just thought we sometime might want to animate from the current value to another value or something similar. And as of now we have no way to do that. But yea might as well only add it when it's needed.

Reply:
Maybe there's a misunderstanding: omitting the getter doesn't delete it, that has to be done explicitly. It just uses the default getter (the one that was previously manually implemented).

Reply:
ohhhh yeah there was a misunderstanding then, my bad.